src/FailureDetectorConfig.md

```diff
@@ -5,6 +5,9 @@ A failure detector is responsible to determine if a member in the cluster is unr
 
 Hazelcast has two built-in failure detectors; Deadline Failure Detector and Phi Accrual Failure Detector.
 
+Since 3.9.1, Hazelcast provides yet another failure detector, ICMP Failure Detector, that if enabled works in parallel with the above ones, but identifies
```

Comment:
Is the name ICMP Failure Detector or Ping Failure Detector? Because the heading of the related section below is Ping Failure Detector.

Reply:
Lets keep it Ping for consistency.